src/pages/ListPage/Filter/Filter.js

```diff
@@ -0,0 +1,298 @@
+              <PriceWrapper>
+                <PriceInput type="text" />
+                <Span margin="5px" right="5px">
+                  _
+                </Span>
+                <PriceInput type="text" />
+              </PriceWrapper>
+            </Modal>
+          )}
+        </div>
+        <div>
+          <Button
+            width="200px"
+            border="1px solid #e4e4e4"
+            height="38px"
+            left="0px"
+            right="20px"
+            onClick={openPriceRange}
+            position="relative"
+          >
+            가격 범위
+          </Button>
+          {isPriceRange && (
+            <Modal position="absolute" size="20px" height="120px" width="265px">
+              <PriceRange>가격 범위</PriceRange>
+              <Price left="3px">최저요금</Price>
+              <Price left="100px">최고요금</Price>
+              <PriceWrapper>
+                <PriceInput type="text" />
+                <Span margin="5px" right="5px">
+                  _
+                </Span>
+                <PriceInput type="text" />
+              </PriceWrapper>
+            </Modal>
+          )}
+        </div>
+        <div>
+          <Button
+            width="200px"
+            border="1px solid #e4e4e4"
+            height="38px"
+            left="0px"
+            right="20px"
+            onClick={openTheme}
+            position="relative"
+          >
+            테마
+          </Button>
+          {isTheme && (
+            <Modal
+              position="absolute"
+              size="20px"
+              display="flex"
+              wrap="wrap"
+              content="space-between"
+              width="200px"
+            >
+              <ThemeTitle> 테마 </ThemeTitle>
+              <Search margin="20px 0 10px 30px">적용하기</Search>
+              {spaceList.map(el => (
+                <Theme key={el.id} el={el} />
+              ))}
+            </Modal>
+          )}
+        </div>
+      </Wrapper>
+      <Center>
+        <Search>SEARCH </Search>
+      </Center>
+    </div>
+  );
+}
+
+const Input = styled.input`
+  border: 1px solid #e4e4e4;
+  width: 200px;
+  height: 36px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 500;
+  padding: 0 10px;
+  margin-left: 10px;
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  height: 60px;
+  border-top: ${props => props.top};
+  border-bottom: 1px solid #e4e4e4;
+`;
+
+const Button = styled.button`
+  padding: 0 15px;
+  width: ${props => props.width};
+  background: ${props => props.background};
+  border: ${props => props.border};
+  height: ${props => props.height};
+  border-radius: 5px;
+  text-align: left;
+  font-size: 14px;
+  font-weight: 500;
+  cursor: pointer;
+  margin: 0 ${props => props.right} 0 ${props => props.left};
+  position: ${props => props.position};
+`;
+
+const Span = styled.span`
+  font-weight: bold;
+  margin-left: ${props => props.margin};
+  margin-right: ${props => props.right};
```

Comment:
```suggestion
  ${({margin, right}) => css`
      margin-left: ${margin};
      margin-right: ${right};
  `};
```
이런식으로도 표현 가능합니다! 참고해보세요

Reply:
문법이 이해가 되지 않아서 반영을 못 시켰습니다 ㅠ

Reply:
지금은 매번 margin-left, margin-right마다 ${} 해서 props를 받아오는 과정을 거치는데 
props를 애초에 한번에 밭아와서 리턴값으로
```
      margin-left: ${margin};
      margin-right: ${right};
```
이런 문자열을 내보내는 방식입니다!
이해 안되시면 오늘 라운지에서 질문주세요!

Reply:
> 지금은 매번 margin-left, margin-right마다 ${} 해서 props를 받아오는 과정을 거치는데 props를 애초에 한번에 밭아와서 리턴값으로
> 
> ```
>       margin-left: ${margin};
>       margin-right: ${right};
> ```
> 
> 이런 문자열을 내보내는 방식입니다! 이해 안되시면 오늘 라운지에서 질문주세요!

시도해봤는데 오류가 나서 내일 여쭤보겠습니다!